src/components/StatusPill.vue

```diff
@@ -61,6 +63,8 @@ const statusColorMap: Record<Status, UIColors> = {
   Paid: 'green',
   Saved: 'blue',
   Submitted: 'blue',
+  Return: 'orange',
```

Comment:
Shouldn't be orange, I'll mention the colors in one of the above files.

Reply:
corrected